code/index.html

```diff
@@ -2,21 +2,220 @@
+        proident, sunt in culpa qui officia deserunt mollit anim id est 
+        laborum. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure 
+        dolor in reprehenderit in voluptate velit esse cillum dolore 
+        eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non 
+        proident, sunt in culpa qui officia deserunt mollit anim id est 
+        laborum. Excepteur sint occaecat cupidatat non 
+        proident, sunt in culpa qui officia deserunt mollit anim id est 
+        laborum. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure 
+        dolor in reprehenderit in voluptate velit esse cillum dolore 
+        eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non 
+        proident, sunt in culpa qui officia deserunt mollit anim id est 
+        laborum. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est 
+        laborum. Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure 
+        dolor in reprehenderit in voluptate velit esse cillum dolore 
+        eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non 
+        proident, sunt in culpa qui officia deserunt mollit anim id est 
+        laborum. xcepteur sint occaecat cupidatat non 
+        proident, sunt in culpa qui officia deserunt mollit anim id est 
+          laborum.</p>
+        </div>
+     </a>
     </section>
 
-    <section class="">
+    <section class="grid-container">
       <!-- The other news  -->
+      <div class="grid-child one">
+        <a href="#">
+          <h3>NEWS</h3>
+          <h6>Posted: August 25th, 2020</h6>
+          <img class="img-small" src="https://images.unsplash.com/photo-1528741254566-d718e868201f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" 
+          alt="Picture with ballons and a pink chair">
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sedu 
+          do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+          laboris nisi ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed 
+          do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco 
+          laboris nisi ut aliquip ex ea commodo consequat. 
+          </p>
+        </a>
+      </div>
+      <div class="grid-child two">
+        <a href="#">
+          <h3>NEWS</h3>
+          <h6>Posted: August 25th, 2020</h6>
+          <img class="img-small" src="https://images.unsplash.com/photo-1597218859878-0e6cad008639?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" 
```

Comment:
If you want to add attribute after line brea as you did with `alt` over here, let's me consequent and do a line break for `class` and `src` as well. It's addressed to all of the occurances of course